Source/WebCore/editing/ApplyStyleCommand.cpp

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2005, 2006, 2008, 2009 Apple Inc. All rights reserved.
+ * Copyright (C) 2005-2022 Apple Inc. All rights reserved.
+ * Copyright (C) 2022 Google Inc. All rights reserved.
```

Comment:
You mean 2014? The patch was authored in 2014, right?

Reply:
Update - My thought process was that if we are merging in 2022 then even if it is old patch then we should use 2022 but now updated. Thanks!